easybuild/easyblocks/t/tensorflow.py

```diff
@@ -1136,10 +1141,12 @@ def install_step(self):
             whl_version = self.version.replace("-rc", "rc")
         else:
             whl_version = self.version
-
-        whl_paths = glob.glob(os.path.join(self.builddir, 'tensorflow-%s-*.whl' % whl_version))
+        whl_dir = self.builddir if LooseVersion(self.version) < '2.16' else (
+            f"{self.builddir}/TensorFlow/tensorflow-{self.version}/bazel-bin/tensorflow/tools/pip_package/wheel_house"
+        )
```

Comment:
uff. That looks more ugly than I anticipated... ;-)
I think this is a bit more readable and easier to extend in case they change their path again:
```suggestion
        if LooseVersion(self.version) < '2.16':
            whl_dir = self.builddir
        else:
            whl_dir = os.path.join(self.start_dir, 'bazel-bin/tensorflow/tools/pip_package/wheel_house')
```

We usually don't hard-code path separators, but I guess it's fine in this case as the alternative would be very verbose.
But at least the parent folder is not hidden this way. Using `start_dir` makes it much shorter